BackendSector/School.js

```diff
@@ -0,0 +1,63 @@
+const Class = require('./Class');
+const Student = require('./Student')
+
+class School {
+  constructor() {
+    this.classes = {
+      "Hellfire" : {
+        name: "Hellfire",
+        teacher: "Remon",
+        students: [{name: "Nilber", age: 15, city: "New York", grade: 50}, {name: "Danielle", age: 15, city: "New York", grade: 100}]
+      } 
+    }
+  }
+  addClass(name, teacher) {
+    let newClass = new Class(name, teacher)
+    if(this.classes[name]){
+        let error = {Status:500, message: "Class already exists at NRES"}
```

Comment:
nitpicking here, status should be lowercase.

Reply:
Fixed, was doing weird stuff